treeherder/model/models.py

```diff
@@ -21,7 +21,6 @@
 from django.forms import model_to_dict
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
-from jsonfield import JSONField
```

Comment:
It looks like this is the last non-migrations usage of the `jsonfield` package, so after the next migrations squashing the package could be removed from the requirements file. Could you file a bug to remind us to remove it?

Also, the comment for the `NoOpSerializer` serializer says that it's only needed due to use of `JSONField`, perhaps we can remove it too?
https://github.com/mozilla/treeherder/blob/bde8e20f1f950c8edb953b1278b33575cc45da72/treeherder/webapp/api/serializers.py#L9-L14
(This is probably best in a separate bug/PR)

Reply:
Sounds good.  Filed Bug 1399999 for this.